homeassistant/components/mill/climate.py

```diff
@@ -197,16 +197,16 @@ class LocalMillHeater(CoordinatorEntity[MillDataUpdateCoordinator], ClimateEntit
     def __init__(self, coordinator: MillDataUpdateCoordinator) -> None:
         """Initialize the thermostat."""
         super().__init__(coordinator)
-        if mac := coordinator.mill_data_connection.mac_address:
```

Comment:
Mac address was not provided with the first firmware version. 
It was added 3 years ago, so it is now unlikely that a device will not provide a mac address.